35-search-insert-position.py

```diff
@@ -0,0 +1,38 @@
+'''
+Given a sorted array and a target value, return the index if the target is found.
+If not, return the index where it would be if it were inserted in order.
+You may assume no duplicates in the array.
+
+Input: [1,3,5,6], 5
+Output: 2
+
+Input: [1,3,5,6], 2
+Output: 1
+
+Input: [1,3,5,6], 7
+Output: 4
+
+Input: [1,3,5,6], 0
+Output: 0
+'''
+
+class Solution:
+  def searchInsert(self, nums, target):
+    left = -1
+    right = len(nums)
+
+    while right - left > 1:
+      middle = (left + right) // 2
```

Comment:
`#pythonrocks` 😉

Reply:
Yeah, I really enjoy it. After struggling with basic stuff like syntax and more advanced shit like list comprehensions, I realize that Python is very well suited for calculations of all sorts and it's incredibly easy to use. I recommend 👍